frontend/src/views/web/challenge/manage.html

```diff
@@ -9,6 +9,14 @@
             </div>
         </div>
     </div>
+
+    <div class="row" ng-if="challenge.isChallengeHost && !challenge.approved_by_admin">
+        <div class="col s12">
```

Comment:
The position of this button has not been fixed. Can you add it below "Toggle Participation"?